src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  margin: 0 auto;
+  padding: 100px $padding;
+  max-width: calendarWidth($gap, $day-width, $padding);
+  gap: $gap;
+
+  &__day {
+    box-sizing: border-box;
+    border: 1px solid black;
+    width: $day-width;
+    height: 100px;
+    transition: transform 0.5s;
```

Comment:
move transition duration to the variable

Reply:
Done. Please clarify why it's required, as the transition option is used only once.